src/graph/test/GoTest.cpp

```diff
@@ -78,7 +78,7 @@ TEST_F(GoTest, OneStepOutBound) {
         cpp2::ExecutionResponse resp;
         auto &player = players_["Boris Diaw"];
         auto *fmt = "GO FROM %ld OVER like "
-                    "| GO FROM $-.id OVER like | GO FROM $-.id OVER serve";
+                    "| GO FROM $-.like_id OVER like | GO FROM $-.like_id OVER serve";
```

Comment:
```$-.alias.prop```
This form looks better to me.

Reply:
Maybe we need discuss this.

Reply:
@dutor @sherman-the-tank @laura-ding

Reply:
I agree with CPWstatic's idea

Reply:
I modified this logic, but there is a problem here. We don't currently support passing "edge.prop/tag.prop" in the pipeline, so we need rename it to use   in the pipeline.